network.go

```diff
@@ -61,7 +61,7 @@ type NetworkController interface {
 	Networks() []Network
 
 	// WalkNetworks uses the provided function to walk the Network(s) managed by this controller.
-	WalkNetworks(walker NetworkWalker)
+	WalkNetworks(walker NetworkWalker) bool
```

Comment:
> Not to add to the interface use case specific requirements.

I think you have missed the point here. This change doesn't add specific use case requirements. The code already exits early if `walker` returns `true`. This change just makes it explicit that this happened, and thereby more convenient to use _in the way that was clearly intended_.